src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* [Breaking] `Enrich` callback is split in to three different callbacks based on
```

Comment:
**Breaking:**

Reply:
could use a better description here. Ideally we should make it easy for end users to just read the changelog and understand what changes they'll need.